apps/ledger-live-desktop/src/renderer/modals/CreateLocalManifest/index.tsx

```diff
@@ -0,0 +1,371 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+
+import React, { useCallback, useMemo, useState } from "react";
+import { Trans } from "react-i18next";
+import Button from "~/renderer/components/Button";
+import Modal, { ModalBody } from "~/renderer/components/Modal";
+import { ScrollArea } from "~/renderer/components/Onboarding/ScrollArea";
+import { Flex } from "@ledgerhq/react-ui";
+import {
+  LiveAppManifest,
+  LiveAppManifestSchema,
+  LiveAppManifestDappSchema,
+  LiveAppManifestDapp,
+  LiveAppManifestParamsNetwork,
+  LiveAppManifestSchemaType,
+} from "@ledgerhq/live-common/platform/types";
+import Text from "~/renderer/components/Text";
+import { useLocalLiveAppContext } from "@ledgerhq/live-common/platform/providers/LocalLiveAppProvider/index";
+import Switch from "~/renderer/components/Switch";
+import FormLiveAppInput from "./FormLiveAppInput";
+import NestedFormCategory from "./NestedFormCategory";
+import updateObjectAtPath from "lodash/set";
+import cloneDeep from "lodash/cloneDeep";
+import FormLiveAppSelector from "./FormLiveAppSelector";
+import FormLiveAppArrayInput from "./FormLiveAppArrayInput";
+import { useCategories, useWalletAPICurrencies } from "@ledgerhq/live-common/wallet-api/react";
+import { WalletAPICurrency } from "@ledgerhq/live-common/wallet-api/types";
+
+import { useManifests } from "@ledgerhq/live-common/platform/providers/RemoteLiveAppProvider/index";
+import FormLiveAppArraySelect from "./FormLiveAppArraySelect";
+import { Separator } from "~/renderer/components/Onboarding/Screens/SelectUseCase/Separator";
+import { DEFAULT_FORM, DEFAULT_VALUES } from "./defaultValues";
+
+export const objectKeys = <Type extends object>(value: Type): Array<keyof Type> => {
+  return Object.keys(value) as Array<keyof Type>;
+};
```

Comment:
Might be good to move this in the helpers file: https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/helpers.ts
And add a link to the source we copied it from (https://github.com/sindresorhus/ts-extras/blob/main/source/object-keys.ts#L24)